app/src/main/res/values/dimens.xml

```diff
@@ -20,4 +20,9 @@
     <dimen name="weight_1">1</dimen>
     <dimen name="sp_50">50sp</dimen>
     <dimen name="dp_20">20dp</dimen>
+    <dimen name="dp_2">2dp</dimen>
+    <dimen name="sp_18">18sp</dimen>
+    <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_40">40dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_40` appears to be unused
 Unused resources make applications larger and slow down builds.